safe-global/web-core
# fix: flag Keystone wallets as hardware wallets + adjust modal `z-index`

## What it solves

Resolves #1110

## How this PR fixes it

Keystone wallets are now marked as hardware wallets and the `z-index` of the Keystone modal container has been increased.

## How to test it

Connect a Keystone wallet to a certain network then switch network. The wallet should be disconnected and the connection window reopened.

All Keystone modals in the Safe should be "above" other elements.
